### ISSUE TITLE
Add HTML <title> element

### ISSUE THREAD

**Issue by evtyz** [loolabs/waterpark#131]
Right now, the title of the webpage is just the URL. See:
![image](https://user-images.githubusercontent.com/58242099/119249298-670e1800-bb65-11eb-93ce-7c86ec0f0da7.png)

There should be an actual `<title>` here, both for accessibility and for SEO.

https://nextjs.org/docs/api-reference/next/head
